engine/src/flutter/impeller/display_list/canvas.cc

```diff
@@ -1017,12 +1017,19 @@ void Canvas::SaveLayer(const Paint& paint,
   // sampling mode.
   ISize subpass_size;
   bool did_round_out = false;
+  Point coverage_origin_adjustment = Point{0, 0};
   if (paint.image_filter) {
     subpass_size = ISize(subpass_coverage.GetSize());
```

Comment:
How come this isn't an issue?

Reply:
I don't fully understand how the coverage computation for image filters are working today. most are pretty destructuve to the exact pixels (and that no one has reported an issue). Given that, I think we should leave well enough alone for now.